README.md

```diff
@@ -1,17 +1,18 @@
-# Thrift Utilities for Elixir
+# A Thrift implementation for Elixir
 
 [![Build Status](https://travis-ci.org/pinterest/elixir-thrift.svg?branch=master)](https://travis-ci.org/pinterest/elixir-thrift)
 [![Coverage Status](https://coveralls.io/repos/pinterest/elixir-thrift/badge.svg?branch=master&service=github)](https://coveralls.io/github/pinterest/elixir-thrift?branch=master)
 
-This package contains a handful of useful utilities for working with
-[Thrift](https://thrift.apache.org/) in Elixir.
+This package contains a complete implementation of [Thrift](https://thrift.apache.org/) for Elixir. It includes a Thrift IDL parser, a code generator, a binary framed client and a binary framed server.
```

Comment:
Good call; I'm using a markdown editor; I'll go over this in Emacs and make sure it's legible.